src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt

```diff
@@ -73,7 +73,11 @@ val RsFunction.abiName: String?
         return stub?.abiName ?: abi?.stringLiteral?.text
     }
 
+/** Returns those function parameters that are not disabled by cfg attributes */
 val RsFunction.valueParameters: List<RsValueParameter>
+    get() = rawValueParameters.filter { it.isEnabledByCfgSelf }
+
```

Comment:
Similarly, something like that would be useful IMO 

```suggestion

/**
 * All function parameters.
 *
 * Should be used in code (PSI) manipulations: intentions, quick-fixes, refactorings, code generation, etc.
 */
```